src/hardware/sblaster.cpp

```diff
@@ -398,6 +398,9 @@ static void configure_sb_filter()
 		sb.chan->EnableZeroOrderHoldUpsampler();
 	};
 
+	// The filter parameters have been tweaked by analysing real hardware
+	// recordings. The results are virtually undistinguishable from the real
```

Comment:
(very minor), maybe use "indistinguishable" (only because it's much more common and translatable for foreign readers?)

Reply:
Ah yeah, my bad... grammar check failed 🛑 🔥